cmd/deployment_test.go

```diff
@@ -251,3 +251,60 @@ func TestDeploymentUserUpdateCommand(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Contains(t, output, expectedOut)
 }
+
+func TestDeploymentAirflowUpgradeCommand(t *testing.T) {
+	testUtil.InitTestConfig()
+	expectedOut := `The upgrade from Airflow 1.10.5 to 1.10.10 has been started.To complete this process, replace the image referenced in your Dockerfile and deploy to Astronomer.`
```

Comment:
- Add space between `d.` and `To`
- Should we say, `To complete this process, add a 1.10.10 image to your Dockerfile and deploy to Astronomer`?

(Assuming we have "desired version" here - if not, then ^this looks good).